checker/examples/units-extension/qual/FrequencyRelations.java

```diff
@@ -34,7 +34,8 @@ public UnitsRelations init(ProcessingEnvironment env) {
     }
 
     /** No multiplications yield Hertz. */
-    public /*@Nullable*/ AnnotationMirror multiplication(AnnotatedTypeMirror lht, AnnotatedTypeMirror rht) {
+    public /*@Nullable*/ AnnotationMirror multiplication(
```

Comment:
There seem to be many changes to formatting. These shouldn't show up, as both sides should already have them. Can you see how to get rid of these differences?